secp256k1-sys/Cargo.toml

```diff
@@ -29,5 +29,6 @@ libc = "0.2"
 default = ["std"]
 recovery = []
 lowmemory = []
+noprecompute = []
```

Comment:
Should we define this as the inverse and say "precompute"? Feature flags in Rust are additive so they should _add_ functionality.

If someone relies on this and adds another dependency to their graph that enables this, their code will break.

Reply:
Done. Thanks.